### PR TITLE
Set width and height on video element

### PR DESCRIPTION
**Pull request by rwd**
Also stop sending specific fields from web resources in API response, and just send the full objects, but whitelist on the web resources component the fields to display.

### CODE REVIEW

File: components/record/WebResources.vue [hunk 1]
```diff
@@ -30,6 +30,11 @@
         type: Array,
         default: () => []
       }
+    },
+    data: function () {
```
Comment (monkemedia) [inline components/record/WebResources.vue]: data () instead of data: function () ?

File: tests/unit/components/record/WebResources.spec.js [hunk 2]
```diff
@@ -15,14 +15,14 @@ describe('components/record/WebResources', () => {
...
-        { rdfAbout: 'http://www.example.org/video.mp4', edmRights: { 'def': [ 'http://creativecommons.org/licenses/by-nc-sa/3.0/' ] } }
+        { about: 'http://www.example.org/image.jpg', dcDescription: ['Art'] },
+        { about: 'http://www.example.org/video.mp4', edmRights: { 'def': [ 'http://creativecommons.org/licenses/by-nc-sa/3.0/' ] } }
       ]
     };
 
     wrapper.setProps(props);
     for (let webResource of props.media) {
-      const webResourceBlock = wrapper.find('#' + cssesc(webResource.rdfAbout, { 'isIdentifier': true }));
+      const webResourceBlock = wrapper.find('#' + cssesc(webResource.about, { 'isIdentifier': true }));
```
Comment (monkemedia) [inline tests/unit/components/record/WebResources.spec.js]: Template literals here instead?

Comment (rwd) [reply tests/unit/components/record/WebResources.spec.js]: Do you mean `isIdentifier: true`?

Comment (rwd) [reply tests/unit/components/record/WebResources.spec.js]: That may need excessive escaping due to the presence of { and } in the code.

**Pull request merged**